client/landing/browsehappy/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from 'calypso/lib/url';
+import Circle from './circle';
+import Logo from './logo-wide';
+
+/**
+ * Style dependencies
+ */
+import 'calypso/assets/stylesheets/style.scss';
+import './style.scss';
+import illustrationURL from './illustration.svg';
+
+const SUPPORTED_BROWSERS_LINK = 'https://wordpress.com/support/browser-issues/#supported-browsers';
+
+export default function Browsehappy( { from } ) {
+	const continueUrl = addQueryArgs( { bypassTargetRedirection: true }, from );
```

Comment:
The server crashes when `from` doesn't exist in the query args (when accessing the page directly)

Reply:
Resolved in 0e03a68